test/FPIS_TEST/CreateStockItemTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using FPIS.Views;
+
+namespace FPIS_TEST
+{
+    [TestClass]
+    public class CreateStockItemTest
+    {
+        [TestMethod]
+        public void ClearFormFields_Test()
+        {
+            CreateStockItem form = new();
+            
+
+            form.materialTextBoxStockItemName.Text = "Some Stock Item";
+            form.ClearFormFields();
+
+            Assert.AreEqual("", form.materialTextBoxStockItemName.Text);
```

Comment:
Since `ClearFormFields` modifies three form fields kindly add the other fields to your Arrange and Assert statement:

Arrange section
```
// Put some value in the unit of measurement
// Put some value in the stock item type
```

Assert section
```
// Assert statement for the unit of measurement
// Assert statement for the stock item type
```